pyston/utils/helpers.py

```diff
@@ -13,7 +13,11 @@
 from .compatibility import FieldDoesNotExist
 
 
-class QuerysetIteratorHelper:
+class IteratorHelper:
```

Comment:
ModelIteratorHelper - aby bylo vidět, že se to vztahuje k modelu

Reply:
Všechny připomínky zapracované.